core/shark_turbine/aot/exporter.py

```diff
@@ -282,9 +283,10 @@ def export(
             {(function_name or "main"): exported_program},
             export_name=module_name or "module",
         )
-    else:
-        assert issubclass(type(mdl), CompiledModule)
+    elif issubclass(type(mdl), CompiledModuleMeta):
```

Comment:
Oh I see it. It would have worked originally if `issubclass(MDL, CompiledModule)`. Either way.

Reply:
Ah, dropping `type()`. Got it.